endaq/batch/core.py

```diff
@@ -71,6 +71,21 @@ def _make_pvss(ch_data_cache):
     return df_pvss.stack(level="axis").reorder_levels(["axis", "frequency (Hz)"])
 
 
+def _make_halfsine_pvss_envelope(ch_data_cache, **kwargs):
+    df_pvss = ch_data_cache._PVSSData.copy()
+    df_pvss["Resultant"] = ch_data_cache._PVSSResultantData
+    df_pvss = df_pvss * endaq.batch.analyzer.MPS_TO_MMPS
```

Comment:
Do we want users to be able to specify other units like inches per second?

Reply:
at this stage I think no? I'd prefer to put that in a separate issue if necessary just to get this feature out there. (also I think this question would apply to a lot of the other functions as well, so a separate issue could cover them all at once.) Do you agree @shanlyMIDE?

Reply:
Yeah I agree with @CFlaniganMide that this would be a nice feature, but also agree with @CrepeGoat that this should be a separate issue/PR later

Reply:
I'm fine leaving it if we're planning on doing it later.

Reply:
added a new issue for this: #129